client/src/js/SM/ReviewsImport.js

```diff
@@ -3021,18 +3030,27 @@ async function showImportResultFile(params) {
             }
 
             async function importReviewArray(reviewArray) {
-                let url = `${STIGMAN.Env.apiBase}/collections/${params.collectionId}/reviews/${params.assetId}`
-                let result = await Ext.Ajax.requestPromise({
-                    url: url,
-                    method: 'POST',
-                    headers: { 'Content-Type': 'application/json;charset=utf-8' },
-                    jsonData: reviewArray
-                })
-                apiReviews = JSON.parse(result.response.responseText)
-                return {
-                    inserted: apiReviews.affected.inserted,
-                    updated: apiReviews.affected.updated,
-                    rejected: apiReviews.rejected
+                if (reviewArray.length < 1) { //Don't try to POST reviews if the review Array is empty.
```

Comment:
Same comment as previous. Would suggest just going with:

```
if (!reviewArray.length) {
  ...
```
this will handle situations where reviewArray is not provided (undefined) or is not an array.

Reply:
and done!